app/scripts/controllers/eventRegistrations.js

```diff
@@ -93,6 +93,32 @@ angular.module('confRegistrationWebApp')
       $scope.queryParameters.block = visibleBlocks;
     };
 
+    // Set initial built in Display Column values
+    var builtInColumnsVisible = {
+      Email: false,
+      Started: false,
+      Completed: false
+    };
+
+    // Used on Display Columns menu click to hide built in columns
+    $scope.hideColumn = function(columnName) {
+      for (var key in builtInColumnsVisible) {
+        if (columnName == key) {
```

Comment:
jshint (JS code style checker) wants a `===` here (which is a safer way of checking if two objects are identical and is causing the [travis build](https://travis-ci.org/CruGlobal/conf-registration-web/builds/144045874#L801-L802) to fail